src/applications/vaos/components/tests/VideoLayoutVA.unit.spec.js

```diff
@@ -33,6 +33,68 @@ describe('VAOS Component: VideoLayoutVA', () => {
     },
   };
 
+  describe('When appointment information is missing', () => {
+    it('should not display heading and text for empty data', async () => {
+      // Arrange
+      const store = createTestStore(initialState);
+      const appointment = {
+        comment: 'This is a test:Additional information',
+        location: {
+          stationId: '983',
+          clinicName: 'Clinic 1',
+          clinicPhysicalLocation: 'CHEYENNE',
+        },
+        videoData: {
+          isVideo: true,
+          facilityId: '983',
+          kind: VIDEO_TYPES.clinic,
+          extension: {
+            patientHasMobileGfe: false,
+          },
+        },
+        vaos: {
+          isCommunityCare: false,
+          isCompAndPenAppointment: false,
+          isCOVIDVaccine: false,
+          isPendingAppointment: false,
+          isUpcomingAppointment: true,
+          isVideo: true,
+          apiData: {},
+        },
+        status: 'booked',
+      };
+
+      // Act
+      const screen = renderWithStoreAndRouter(
+        <VideoLayoutVA data={appointment} />,
+        {
+          store,
+        },
+      );
+
+      // Assert
+      expect(
+        screen.getByRole('heading', {
+          level: 1,
+          name: /Video appointment at VA location/i,
+        }),
+      );
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /Who/i,
+        }),
+      ).not.to.exist;
+
+      expect(
+        screen.queryByRole('heading', {
+          level: 2,
+          name: /What/i,
+        }),
+      ).not.to.exist;
```

Comment:
Need to add assertions for missing facility and clinic info. Also consider adding another case where facility id is present but missing facility info.

Reply:
@JunTaoLuo I've added tests cases for this (the facility id/missing facility scenario) for `ClaimExamLayout.jsx`, `VideoLayoutVA.jsx` and `InPersonLayout.jsx`